core/src/components/button/button.ionic.scss

```
@@ -102,44 +101,24 @@
 // Button Sizes
 // -------------------------------------------------------------------------------
 
-/* Extra Small Button */
-:host(.button-xsmall) {
-  --padding-top: #{globals.$ion-space-100};
-  --padding-end: #{globals.$ion-space-300};
-
-  min-height: globals.$ion-space-600;
-
-  font-size: globals.$ion-font-size-300;
-}
-
 /* Small Button */
 :host(.button-small) {
   --padding-top: #{globals.$ion-space-200};
   --padding-end: #{globals.$ion-space-400};
 
-  min-height: #{globals.$ion-scale-800};
+  @include globals.typography(globals.$ion-body-action-sm);
```

```suggestion
  @include globals.typography(globals.$ion-body-action-md);
```

I see `body/action/md` on the small button

---

I cheked with UX team and it should be sm for small, md for default and lg for large!